server/services/bookingService.ts

```diff
@@ -144,6 +144,18 @@ export default class BookingService {
       )
     }
 
+    if (booking.extensions.length) {
+      const collatedNotes = booking.extensions
+        .map(extension => extension.notes)
+        .filter(notes => notes?.length > 0)
+        .join('\n\n')
+
+      rows.push({
+        key: this.textValue('Extension notes'),
+        value: this.htmlValue(formatLines(collatedNotes)),
+      })
+    }
```

Comment:
Again, this is not somehting to act on now, but I've started to move this kind of UI-first work to util functions, which we can then use in the views themselves. It makes it easier to test and seperates the data and presentation later a bit better.

Reply:
Ah, yep, saw that. Yeah, may well follow that...